tests/unit_tests/gui/test_main_window.py

```diff
@@ -356,18 +361,9 @@ def handle_add_dialog():
     manage_tool.trigger()
 
 
-def test_that_the_manual_analysis_tool_works(
-    ensemble_experiment_has_run, opened_main_window, qtbot, run_experiment
-):
-    """This runs a full manual update workflow, first running ensemble experiment
-    where some of the realizations fail, then doing an update before running an
-    ensemble experiment again to calculate the forecast of the update.
-    """
-    gui = opened_main_window
-    analysis_tool = gui.tools["Run analysis"]
-
+def _handle_analysis_dialog(analysis_tool, qtbot):
     # Open the "Run analysis" tool in the main window after ensemble experiment has run
-    def handle_analysis_dialog():
+    def do_the_handle():
```

Comment:
I think want to stick to the convention of dialog handlers being put inline. If we start mixing how this is done, its going to be very difficult to follow

Reply:
then i will have to add a pylint ignore statement because the function from which i pulled the handler had too many statements

Reply:
removed linting fix